File: 0682-baseball-game/0682-baseball-game.py
class Solution:
    def calPoints(self, operations: List[str]) -> int:
        ans=[]
        a=0
        for i in operations:
            if i=="C":
                a=ans.pop()
            elif i=="D":
                ans.append(2*ans[-1])
            elif i=="+":
                ans.append(ans[-1]+ans[-2])
            else:
                ans.append(int(i))
        return sum(ans)

File: 0682-baseball-game/0682-baseball-game.kt
class Solution {
fun calPoints(operations: Array<String>): Int {
    var stack = ArrayList<Int>()
    operations.forEach {
        when (it) {
            "+" -> {
                stack.add(stack[stack.size - 1] + stack[stack.size - 2])
            }
            "D" -> {
                stack.add(stack[stack.size - 1] * 2)
            }
            "C" -> {
                stack.removeAt(stack.size - 1)
            }
            else -> {
                stack.add(it.toInt())
            }
        }
    }
    return stack.sum()
}
}